MOPTools/MOP_Discovery/query_operations/queryendpoint.py

```diff
@@ -1,4 +1,15 @@
-SPARQL_ENDPOINTS = {
-    'ontospecies': 'http://theworldavatar.com/blazegraph/namespace/ontospecies/sparql',
-    'ontomops': 'https://kg.cmclinnovations.com/blazegraph_geo/namespace/ontomops/sparql'
```

Comment:
This does not look like what you put into the config file...

Reply:
As discussed, these endpoints are no longer working and are replaced with the ones in the config.